docassemble/MotionToStayEviction/data/questions/review_screen.yml

```diff
@@ -0,0 +1,386 @@
+---
+event: review_case_info
+id: |-
+  review screen
+question: |-
```

Comment:
I had to look up what these were (`|-` is "no new line at end (strip), default `|` is  "single newline at end). Not sure what practical difference it makes though? I'm fine with it, just curious about your motivations.

Reply:
Just the YAML library used in the review screen generator